bucket/jadx.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.1.0",
+    "description": "Dex to Java decompiler",
+    "homepage": "https://github.com/skylot/jadx",
+    "license": "Apache-2.0",
+    "url": "https://github.com/skylot/jadx/releases/download/v1.1.0/jadx-1.1.0.zip",
+    "hash": "91948067a60feee36512e043b288f726678ddd760aa11a01496b53df95465b16",
+    "bin": [
+        "bin\\jadx.bat",
+        "bin\\jadx-gui.bat"
```

Comment:
What are available parameters?

Reply:
You can see the available parameters via "https://github.com/skylot/jadx#usage", but usually the default parameters are sufficient.

Reply:
> What are available parameters?

You can see the available parameters via "https://github.com/skylot/jadx#usage", but usually the default parameters are sufficient.